GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -476,16 +503,21 @@ private ProcessResult RunFastFetch(string args)
         {
             args = args + " --verbose";
 
-            string fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "fastfetch.exe");
-            if (!File.Exists(fastfetch))
+            string fastfetch;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "netcoreapp2.1", "fastfetch.dll");
+            }
+            else
             {
-                fastfetch = "fastfetch.exe";
+                fastfetch = Path.Combine(Settings.Default.CurrentDirectory, "fastfetch.dll");
             }
 
+            File.Exists(fastfetch).ShouldBeTrue();
             Console.WriteLine($"Using {fastfetch}");
 
-            ProcessStartInfo processInfo = new ProcessStartInfo(fastfetch);
-            processInfo.Arguments = args;
+            ProcessStartInfo processInfo = new ProcessStartInfo("dotnet");

```

Comment:
Why are the tests running against "fastfetch.dll"? Can we point them to the published FastFetch?

Reply:
On MacOS, we run the FunctionalTests out of the Publish folder, so we can easily find the published FastFetch executable. On Windows, we're currently running the FunctionalTests out of the BuildOutput folder. Instead of reworking how we run tests on Windows at this time, I went for the cheap fix of running FastFetch via `dotnet`. Whenever we move the rest of the project to .NET Core (or extract FastFetch into its own codebase), this can be cleaned up.

Reply:
Could you file a follow up issue to run the tests against the exe?  In general we've tried to run the functional tests as black-box tests against the final binaries.

Reply:
#624